.github/workflows/build.yml

```diff
@@ -0,0 +1,16 @@
+name: Build test
+on:
+  push:
+    branches:
+      - main
```

Comment:
@AbhiKul10 this should be for a Pull request.

Reply:
@Wathsara changed it for pull request, but I have a doubt like when changing it for pull_request, are we required to mention the branch also?

Reply:
@AbhiKul10 Yes we have to